src/ManifestGenerator/manifestgen.py

```diff
@@ -0,0 +1,162 @@
+#!/usr/bin/env python3
+
+import argparse
+import csv
+import io
+import json
+import dataclasses
+import sys
+import typing
+import random
+import boto3
```

Comment:
Note that in the original version, I had a `requirements.txt`, should we have that as well, do we need a `setup.py` or equivalent?

Reply:
the new README.md file i created includes the ## Requirements section. i interpreted this `requirements.txt` to be just a text file and that it was not really required for script execution. 

regarding `setup.py` i do not see it for other utilities on this repo so i am guessing its not required.

Reply:
Looks like we have it here https://github.com/awslabs/amazon-redshift-utils/blob/master/src/requirements.txt
It should be fine then